resources/import.php

```diff
@@ -305,28 +309,31 @@ function searchForShortName($shortName, $array)
 							}
 						}
 					}
-					foreach ($allIsbnOrIssn_columns as $columnCriterion)
+					if (!empty($allIsbnOrIssn_columns))
 					{
-						if($columnCriterion['delimiter'] !== '')
+						foreach ($allIsbnOrIssn_columns as $columnCriterion)
 						{
-							$columnValues = explode($columnCriterion['delimiter'],$data[$columnCriterion['column']]);
-							foreach($columnValues as $value)
+							if($columnCriterion['delimiter'] !== '')
 							{
-								if($value != '')
+								$columnValues = explode($columnCriterion['delimiter'],$data[$columnCriterion['column']]);
+								foreach($columnValues as $value)
 								{
-									$isbnIssn_values[] = $value;
+									if($value != '')
+									{
+										$isbnIssn_values[] = $value;
+									}
 								}
 							}
-						}
-						else
-						{
-							if($data[$columnCriterion['column']] != '')
+							else
 							{
-								$isbnIssn_values[] = $data[$columnCriterion['column']];
+								if($data[$columnCriterion['column']] != '')
+								{
+									$isbnIssn_values[] = $data[$columnCriterion['column']];
+								}
 							}
 						}
 					}
-					$deduping_count = count($resourceObj->getResourceByIsbnOrISSN($deduping_values));
+					$deduping_count = isset($deduping_values) ? count($resourceObj->getResourceByIsbnOrISSN($deduping_values)) : 0;
```

Comment:
I guess some alternatives would be adding an `else` for when there is no `$dedupeCriterion['column']` or maybe even initializing an empty `$deduping_values` array before it is needed.